enms.py

```diff
@@ -1,6 +1,16 @@
 from eNMS import create_app, db
 from flask_migrate import Migrate
 from os.path import abspath, dirname
+import os
+
 
 app = create_app(abspath(dirname(__file__)))
 Migrate(app, db)
+
+if '__main__' == __name__:
+    try:
+        if os.environ['FLASK_APP'] is not 'enms.py':
+            os.environ['FLASK_APP'] = 'enms.py'
+    except KeyError:
+        os.environ['FLASK_APP'] = 'enms.py'
+    app.run(host='0.0.0.0', port='5000')
```

Comment:
This method for starting a Flask application is deprecated, see https://softwareengineering.stackexchange.com/questions/326517/why-is-flask-cli-recommended-over-flask-run. Is there a reason why you added these lines ? Even if it wasn't deprecated, couldn't you do the same thing in one single line `os.environ['FLASK_APP'] = 'enms.py'` ?

Reply:
There is a paragraph in the readme about how to start the app with `flask run` : https://github.com/afourmy/eNMS#getting-started